src/nodes.coffee

```diff
@@ -3097,12 +3211,13 @@ exports.If = class If extends Base
 
 UTILITIES =
   modulo: -> 'function(a, b) { return (+a % (b = +b) + b) % b; }'
-
+  objectWithoutKeys: -> 'function(obj, props) { return Object.keys(obj).reduce(function(a,c) { return (![].includes.call(props, c)) && (a[c] = obj[c]), a; }, {}); }'
```

Comment:
I'm not sure if utilities can use other utilities, since they have to way of injecting them (e.g. if some code uses `objectWithoutKeys` but *doesn't* use `hasProp`, `hasProp` won't be in the source.